### ISSUE TITLE
`git@github.com' is not a git command` on installing with `core.autocrlf=false`

### ISSUE THREAD

**Issue by germn** [git-for-windows/git#2643]
**TLDR** If on installing you choose autocrlf option "checkout as-is, commit as-is" (`core.autocrlf=false`), then if you try to clone any repo by ssh (for example, `git clone git@github.com:pallets/flask.git`) you'll get an error:

> `'git@github.com' is not a git command`

Bug seems to be [rather old](https://gitlab.com/tortoisegit/tortoisegit/-/issues/1968).
I could reproduce it on git versions: 2.25, 2.26, 2.27rc2.
Reinstalling git with default autocrlf option and then running `git config --global core.autocrlf false` solves the problem and leads to desired git behavior.

---

- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options

git version 2.26.2.windows.1
cpu: x86_64
built from commit: 4267c88d0d8936ae723bba55e0576b1c4f3d4e1a
sizeof-long: 4
sizeof-size_t: 8
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

Microsoft Windows [Version 10.0.18363.836]
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

Editor Option: Notepad++
Custom Editor Path:
Path Option: Cmd
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: OpenSSL
CRLF Option: CRLFCommitAsIs
Bash Terminal Option: MinTTY
Performance Tweaks FSCache: Enabled
Use Credential Manager: Disabled
Enable Symlinks: Enabled
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

-

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

ConEmu (but problem reproducable in CMD, Bash, etc.)

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
git clone git@github.com:pallets/flask.git
```
(reproducible on any repo)

 - What did you expect to occur after running these commands?

Repo will be cloned.

 - What actually happened instead?

I got an errro with following text:

> `'git@github.com' is not a git command`

**Comment by dscho** [comment]
>  I could reproduce it on git versions: 2.25, 2.26, 2.27rc2.

I cannot reproduce it. I tried in Git Bash. I did set `Checkout as-is, commit as-is`:

```sh
$ git config --show-scope  core.autocrlf
system  false
```

The TortoiseGit issue you referenced seems to have resolved itself by deleting incorrect TortoiseGit-related registry settings.

**Comment by germn** [comment]
I tried it on fresh Windows install and couldn't reproduce.
Can't think of something significant that differed first attempt from the second one.
Since no one had such problem for a long time, I'll close the issue until I find a way to reproduce it.
